Source/Core/Core/HW/DVDInterface.h

```diff
@@ -91,6 +93,12 @@ void DoState(PointerWrap &p);
 
 void RegisterMMIO(MMIO::Mapping* mmio, u32 base);
 
+// Direct disc access
+const DiscIO::IVolume& GetVolume();
+bool SetVolumeName(const std::string& _rFullPath);
```

Comment:
you can get rid of the _r prefixes and just lowercase the parameters here.

e.g.

```cpp
bool SetVolumeName(const std::string& full_path);
```